libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/Customization.kt

```diff
@@ -0,0 +1,65 @@
+package com.mapbox.navigation.dropin
+
+import com.mapbox.navigation.dropin.binder.UIBinder
+import com.mapbox.navigation.ui.maps.route.arrow.model.RouteArrowOptions
+import com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions
+
+/**
+ * DropInNavigationView customizations.
+ */
+class Customization {
+
+    /**
+     * Customize the speed limit view by providing your own [UIBinder].
+     * Use [UIBinder.USE_DEFAULT] to reset to default.
+     */
+    var speedLimit: UIBinder? = null
+
+    /**
+     * Customize the maneuvers view by providing your own [UIBinder].
+     * Use [UIBinder.USE_DEFAULT] to reset to default.
+     */
+    var maneuver: UIBinder? = null
+
+    /**
+     * Customize the road name view by providing your own [UIBinder].
+     * Use [UIBinder.USE_DEFAULT] to reset to default.
+     */
+    var roadName: UIBinder? = null
+
+    /**
+     * Customize the Info Panel Trip Progress by providing your own [UIBinder].
+     * Use [UIBinder.USE_DEFAULT] to reset to default.
+     */
+    var infoPanelTripProgressBinder: UIBinder? = null
+
+    /**
+     * Customize the Info Panel Header by providing your own [UIBinder].
+     * Use [UIBinder.USE_DEFAULT] to reset to default.
+     */
+    var infoPanelHeaderBinder: UIBinder? = null
+
+    /**
+     * Customize the Info Panel Content by providing your own [UIBinder].
+     * Use [UIBinder.USE_DEFAULT] to reset to default.
+     */
+    var infoPanelContentBinder: UIBinder? = null
+
+    /**
+     * Customize the Action Buttons by providing your own [UIBinder].
+     * Use [UIBinder.USE_DEFAULT] to reset to default.
+     */
+    var actionButtonsBinder: UIBinder? = null
+
+    /**
+     * Provide custom route arrow options. This must be set before your
+     * activity or fragment's onStart() in order to take effect.
+     */
+    var routeArrowOptions: RouteArrowOptions? = null
+
+    /**
+     * Provide custom route line options. This must be set before your
+     * activity or fragment's onStart() in order to take effect.
+     */
+    var routeLineOptions: MapboxRouteLineOptions? = null
```

Comment:
I think it will be good to just customize the view binders here. Drop-In UI in general will have lots of options. This is just the beginning. I would like us to rename this class to something more specific like `ViewBinderCustomization` and remove both `routeArrowOptions` and `routeLineOptions`. We can expose passing the options differently in a separate PR.